pkg/controller/certs/certs.go

```diff
@@ -100,9 +124,6 @@ func GenerateCA(notAfter time.Time) (*KeyPair, error) {
 // CreateSignedServingPair creates a serving cert/key pair signed by the given ca
 func CreateSignedServingPair(notAfter time.Time, ca *KeyPair, hosts []string) (*KeyPair, error) {
 	notBefore := time.Now()
-	if notAfter.Before(notBefore) {
```

Comment:
why did we stop checking this?

Reply:
I wanted to be able to create bad serving cert pairs for testing purposes. I thought this would be a reasonable approach since we were thinking about converting to the serving cert signing service in the future anyway.

Reply:
Make a function in a test file that generates bad certs? let's err on the side of more sanity checks for prod code

Reply:
got it